homeassistant/components/flux_led/light.py

```diff
@@ -279,32 +277,15 @@ def turn_on(self, **kwargs):
         if not self.is_on:
             self._bulb.turnOn()
 
-        hs_color = kwargs.get(ATTR_HS_COLOR)
-
-        if hs_color:
-            rgb = color_util.color_hs_to_RGB(*hs_color)
-        else:
-            rgb = None
-
         brightness = kwargs.get(ATTR_BRIGHTNESS)
         effect = kwargs.get(ATTR_EFFECT)
-        white = kwargs.get(ATTR_WHITE_VALUE)
-        color_temp = kwargs.get(ATTR_COLOR_TEMP)
-
-        # handle special modes
-        if color_temp is not None:
-            if brightness is None:
-                brightness = self.brightness
-            if color_temp > COLOR_TEMP_WARM_VS_COLD_WHITE_CUT_OFF:
-                self._bulb.setRgbw(w=brightness)
-            else:
-                self._bulb.setRgbw(w2=brightness)
-            return
```

Comment:
Removed because the light does not announce color_temperature support, so `ATTR_COLOR_TEMP` will be removed by the base entity's service handler.
Does flux_led actually support color temperature in some cases?

Reply:
Yes, flux_led supports color temp.  Below are the color modes it's able to handle.  

COLOR_MODE_ONOFF
COLOR_MODE_BRIGHTNESS
COLOR_MODE_COLOR_TEMP
COLOR_MODE_RGB
COLOR_MODE_RGBW
COLOR_MODE_RGBWW